src/views/v2/list/month-separator.php

```diff
@@ -13,7 +13,8 @@
  *
  */
 $month = $this->get( 'month' );
+$year  = $this->get( 'year' )
 ?>
 <div class="tribe-events-calendar-list__separator-month">
-	<time class="tribe-events-calendar-list__separator-month-text tribe-common-b1 tribe-common-b1--bold" datetime="1970-01-01T00:00:00+00:00"><?php echo esc_html( $month ); ?></time>
+	<time class="tribe-events-calendar-list__separator-month-text tribe-common-b1" datetime="1970-01-01T00:00:00+00:00"><?php echo esc_html( $month ); ?> <?php echo esc_html( $year ); ?></time>
```

Comment:
Break into multiple lines
And lets make sure the `datetime=""` is generated dynamically.